etip/trackers/tests.py

```diff
@@ -290,10 +348,75 @@ def test_with_search_query_with_results(self):
         self.assertNotContains(response, tracker_2.name)
         self.assertEqual(response.context['count'], 1)
 
+    def test_with_only_collisions_filter(self):
+        tracker_1 = Tracker(
+            name='match_name_tracker_1',
+            code_signature='toto.com',
+            network_signature='network_1',
+            website='https://website1'
+        )
+        tracker_2 = Tracker(
+            name='random name',
+            code_signature='tracker_code_2',
+            network_signature='network_2',
+            website='https://website2'
+        )
+        tracker_3 = Tracker(
+            name='tracker 3',
+            code_signature='toto.com',
+            network_signature='network.signature',
+            website='https://website3'
+        )
+        tracker_4 = Tracker(
+            name='tracker 4',
+            code_signature='code_4',
+            network_signature='network.signature',
+            website='https://website4'
+        )
+
+        tracker_1.save()
+        tracker_2.save()
+        tracker_3.save()
+        tracker_4.save()
+
+        c = Client()
+        response = c.get('/', {'only_collisions': 'true'})
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, tracker_1.name, 2)
+        self.assertContains(response, tracker_3.name, 3)
+        self.assertContains(response, tracker_4.name, 2)
+        self.assertNotContains(response, tracker_2.name)
+        self.assertEqual(response.context['count'], 3)
+
+    def test_with_only_collisions_filter_without_collisions(self):
+        tracker_1 = Tracker(
+            name='match_name_tracker_1',
+            code_signature='toto.com',
+            network_signature='network_1',
+            website='https://website1'
+        )
+        tracker_2 = Tracker(
+            name='random name',
+            code_signature='tracker_code_2',
+            network_signature='network_2',
+            website='https://website2',
+        )
+
+        tracker_1.save()
+        tracker_2.save()
+
+        c = Client()
+        response = c.get('/', {'only_collisions': 'true'})
+        self.assertEqual(response.status_code, 200)
+        self.assertNotContains(response, tracker_1.name)
+        self.assertNotContains(response, tracker_2.name)
+        self.assertEqual(response.context['count'], 0)
+
     def test_with_results_and_paginate(self):
         for i in range(0, 25):
             Tracker(
-                name='AcTracker_name'
+                name='AcTracker_name',
+                code_signature='toto.com'
```

Comment:
As only_collisions is set to false by default, why did you made changes to this test?
I think I would let it the way it was before.

Reply:
I've also add the parameter to the query in order to be sure that both parameters will not break the pagination. :thinking:

Reply:
Hmm, interesting, I didn't think about that.
I'm wondering if that would make sense to have 2 tests, one with and one without the option. WDYT?

Reply:
I've add another test :)